molgenis-data/src/main/java/org/molgenis/data/meta/EntityTypeRepositoryDecorator.java

```diff
@@ -116,7 +116,8 @@ private void addEntityType(EntityType entityType)
 	{
 		// add row to entities table
 		delegate().add(entityType);
-		if (!entityType.isAbstract() && !MetaDataService.isMetaEntityType(entityType))
+		if (!entityType.isAbstract() && !MetaDataService.isMetaEntityType(
+				entityType)) // TODO can we remove !MetaDataService.isMetaEntityType(entityType)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)